ui/scripts/configuration.js

```diff
@@ -2939,7 +2940,41 @@
                                         label: 'label.cache.mode'
                                     },
                                     tags: {
-                                        label: 'label.storage.tags'
+                                        label: 'label.storage.tags',
+                                        docID: 'helpPrimaryStorageTags',
+                                        isEditable: true,
+                                        isTokenInput: true,
+                                        dataProvider: function(args) {
+                                            $.ajax({
+                                                url: createURL("listStorageTags"),
+                                                dataType: "json",
+                                                success: function(json) {
+                                                    var item = json.liststoragetagsresponse.storagetag;
```

Comment:
@RodrigoDLopez does this require any changes in Primate?
Can you check and send a PR to Primate? Thnx

Reply:
Hi @rhtyd I believe yes, some changes will be necessary on Cloud-Primate
I will take a look on this project, but i'm not familiarized with VUE.js